doc/doc/advanced/incremental.md

```diff
@@ -89,23 +89,23 @@ It will do the following:
 ### Incremental Phase
 
 In the incremental phase, suppose you added feature 2 to your udf, `ext_has_spouse_features.f2.py`.
-Let's say you saved this modified udf in a file named `ext_has_spouse_features.f2.ddl`.
+Let's say you saved this modified udf in a file named `ext_has_spouse_features.f2.ddlog`.
 
 <script src="https://gist-it.appspot.com/github.com/HazyResearch/deepdive/blob/master/examples/spouse_example/postgres/incremental/udf/ext_has_spouse_features.f2.py?footer=minimal&slice=27:39">
 </script>
 
 
 You need to mark in the DDlog program which udf has been modified to produce the correct incremental pipeline.
-Let's say you saved this modified DDlog program in a file named `spouse_example.f2.ddl`.
+Let's say you saved this modified DDlog program in a file named `spouse_example.f2.ddlog`.
 
-<script src="https://gist-it.appspot.com/github.com/HazyResearch/deepdive/blob/master/examples/spouse_example/postgres/incremental/spouse_example.f2.ddl?footer=minimal&slice=67:70">
+<script src="http://gist-it.appspot.com/https://github.com/HazyResearch/deepdive/blob/master/examples/spouse_example/incremental/spouse_example.f2.ddlog?footer=minimal&slice=67:70">
```

Comment:
Make sure the line numbers are in sync with the modified code (67-70 probably moved)